### PR TITLE
feat(logging): group logging in github by task id

### PR DESCRIPTION
**Pull request by mehulkar**
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines

### CODE REVIEW

File: cli/internal/run/real_run.go [hunk 1]
```diff
@@ -305,24 +305,24 @@ func (ec *execContext) exec(ctx gocontext.Context, packageTask *nodes.PackageTas
 	tracer(runsummary.TargetBuilding, nil, &successExitCode)
 
 	var prefix string
-	var prettyPrefix string
 	if ec.rs.Opts.runOpts.LogPrefix == "none" {
 		prefix = ""
+	} else if ec.rs.Opts.runOpts.LogPrefix == "github" {
+		prefix = fmt.Sprintf("::group::%s ", packageTask.OutputPrefix(ec.isSinglePackage))
```
Comment (mehulkar) [inline cli/internal/run/real_run.go:311]: not sure if i need to remove the "pretty" (i.e. colors) for the github prefix

**Comment by mehulkar** [comment]
The e2e windows tests are failing because it can't find the turbo binary. I want to say this is an issue because `cli#build` is getting a cache hit and the restore isn't working as expected on windows?

**Comment by weyert** [comment]
Similar functionality exists for other popular CI solution Teamcity:
https://www.jetbrains.com/help/teamcity/9.0/build-script-interaction-with-teamcity.html

e.g. `##teamcity[blockOpened name='<blockName>' description='<this is the description of blockName>']`

Another popular CI solution that supports something similar Gitlab:
https://docs.gitlab.com/ee/ci/jobs/#custom-collapsible-sections

**Comment by mehulkar** [comment]
@weyert thanks for the note! do you mind making new issues for these if you're interested in adding support? I am on leave for a few weeks, so this PR will likely just add Github support. (You're also welcome to make a PR if you like, of course!)

**Comment by gsoltis** [comment]
Closing in favor of https://github.com/vercel/turbo/pull/5385

**Pull request closed without merging**